angel_system/data/common/load_data.py

```diff
@@ -174,14 +177,76 @@ def activities_from_dive_csv(filepath: str) -> List[Activity]:
                 frame,
                 a.conf,
             )
+
     # Assert that all activities have been assigned an associated end time.
     assert np.inf not in {a.end for a in id_to_activity.values()}, (
         f"Some activities in source CSV do not have corresponding end time "
         f"entries: {filepath}"
     )
+
     return list(id_to_activity.values())
 
 
+def add_inter_steps(
+    gt, min_start_time, max_end_time, add_inter_steps=True, add_before_after_steps=True
```

Comment:
`ad_inter_steps` is both the name of the function *and* a parameter name. Consider an alternative parameter name.